core/db/migrate/20210512191732_create_spree_cms_pages.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSpreeCmsPages < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_cms_pages do |t|
+      t.column :title, :string, null: false
+      t.column :meta_title, :string
+      t.column :content, :text
+      t.column :meta_description, :text
+      t.column :visible, :boolean, default: false
+      t.column :slug, :string
```

Comment:
@MatthewKennedy let's create on index on `slug` and another index on `slug` + `store_id` + `locale`